apps/web/modules/ee/license-check/lib/utils.ts

```diff
@@ -81,7 +81,14 @@ const fetchLicenseForE2ETesting = async (): Promise<{
       // first call
       const newResult = {
         active: true,
-        features: { isMultiOrgEnabled: true, twoFactorAuth: true, sso: true, contacts: true, projects: 3 },
+        features: {
+          isMultiOrgEnabled: true,
+          twoFactorAuth: true,
+          sso: true,
+          contacts: true,
+          projects: 3,
+          whitelabel: true,
```

Comment:
We made a last minute change to the way we customize licenses. Can you please add a `removeBranding` flag to the ee server for removing the branding?
We will still use the `whitelabel` flag for the whitelabel 2 ticket.